x-pack/solutions/security/plugins/security_solution/server/endpoint/services/actions/clients/endpoint/endpoint_actions_client.ts

```diff
@@ -64,6 +64,12 @@ const getInvalidAgentsWarning = (invalidAgents: string[]) =>
 export class EndpointActionsClient extends ResponseActionsClientImpl {
   protected readonly agentType: ResponseActionAgentType = 'endpoint';
 
+  protected async fetchAgentPolicyInfo(
```

Comment:
Please double check whether this method is actually used anywhere. From what I can see, it's not called within this class, and the class itself doesn't appear to be extended or used elsewhere either.

Reply:
Thats correct - its not called within this sub-class. `fetchAgentPolicyInfo()` is an abstract method in the base class which much be implemented by each sub-class. It is used when the response action request is being written to the index.